river/view_stack.zig

```diff
@@ -108,6 +110,50 @@ pub fn ViewStack(comptime T: type) type {
             }
         }
 
+        /// Swap the nodes one and two.
+        /// pointers to Node.T will point to the same data as before
+        pub fn swap(self: *Self, one: *Node, two: *Node) void {
```

Comment:
Style: `one` and `two` are not very good variable names IMO. For such functions, `a` and `b` are more common.